projects/cmake/build.sh

```diff
@@ -108,6 +110,10 @@ if [ -n "$boost_lib" ] ; then
     cmake_args="-DBOOST_LIBRARYDIR=\"${boost_lib}\" $cmake_args"
 fi
 
+if [ -n "$boost_include" ] ; then
+    cmake_args="-DBOOST_INCLUDEDIR=${boost_include} $cmake_args"
```

Comment:
As a separate, but related, point to the one above, I think we should set `Boost_INCLUDE_DIR` directly, instead of setting BOOST_INCLUDEDIR, and then setting `Boost_INCLUDE_DIR` later.

Reply:
Argh... this seems a little bit complicated: https://android.googlesource.com/platform/prebuilts/cmake/windows-x86/+/refs/heads/studio-1.3-dev/share/cmake-3.2/Modules/FindBoost.cmake

Reply:
I guess cmake has its own versions of the normal boost variables. I was trying to be consistent and extra safe to make sure that the intended paths would be used in the build process.